tfhe/src/core_crypto/gpu/algorithms/test/mod.rs

```diff
@@ -4,8 +4,15 @@ mod glwe_sample_extraction;
 mod lwe_keyswitch;
 mod lwe_linear_algebra;
 mod lwe_multi_bit_programmable_bootstrapping;
+mod lwe_packing_keyswitch;
 mod lwe_programmable_bootstrapping;
 
+pub struct CudaPackingKeySwitchKeys<Scalar: UnsignedInteger> {
+    pub lwe_sk: LweSecretKey<Vec<Scalar>>,
+    pub glwe_sk: GlweSecretKey<Vec<Scalar>>,
+    pub pksk: CudaLwePackingKeyswitchKey<Scalar>,
+}
+
```

Comment:
this is very weird, it keeps secret and public keys together, why is this needed ?

Reply:
It is not needed. I just copied CPU's test for packing keyswitch at https://github.com/zama-ai/tfhe-rs/blob/main/tfhe/src/core_crypto/algorithms/test/lwe_packing_keyswitch.rs

Reply:
oh my bad, did not see it was a test struct !